com.unity.netcode.gameobjects/Runtime/SceneManagement/NetworkSceneManager.cs

```diff
@@ -1994,8 +1996,29 @@ private void OnClientLoadedScene(uint sceneEventId, Scene scene)
         /// synchronized.
         /// </summary>
         /// <param name="clientId">newly joined client identifier</param>
-        internal void SynchronizeNetworkObjects(ulong clientId)
+        internal void SynchronizeNetworkObjects(ulong clientId, bool synchronizingService = false)
```

Comment:
Can we add a code doc `<param />`  explaining this parameter?

Reply:
Ahhh, yeah... good catch!